_ingest-pipelines/processors/ml-inference.md

```diff
@@ -94,6 +113,31 @@ PUT /_ingest/pipeline/ml_inference_pipeline
 ```
 {% include copy-curl.html %}
 
+For externally hosted models, in a Predict API request, all fields are usually nested inside the `parameters` object:
+
+```json
+POST /_plugins/_ml/models/cleMb4kBJ1eYAeTMFFg4/_predict
+{
+  "parameters": {
+    "input": [
+      {
+        ...
+      }
+    ]
+  }
+}
+```
+
+When specifying the `input_map` for an externally hosted model, you can directly reference the `input` field  instead of providing its dot path `parameters.input`:
```

Comment:
```suggestion
When specifying the `input_map` for an externally hosted model, you can directly reference the `input` field instead of providing its dot path `parameters.input`:
```